gui/mozregui/pref_editor.py

```diff
@@ -30,13 +30,13 @@ def data(self, index, role=Qt.DisplayRole):
             if index.column() == 0:
                 return name
             else:
-                return repr(value)
+                return QString(value).__str__()
 
     def flags(self, index):
         return Qt.ItemIsSelectable | Qt.ItemIsEditable | Qt.ItemIsEnabled
 
     def setData(self, index, new_value, role=Qt.EditRole):
-        new_value = str(new_value.toString())
+        new_value = new_value.toPyObject().__str__()
```

Comment:
Oh cool, if that is a unicode I'm not agains it. I find unicode() easier to read, but I'm ok with the other notation (maybe with a comment in this case to say that it is real unicode).